crates/ra_syntax/test_data/parser/err/0044_bad_use_tree_list.rs

```diff
@@ -0,0 +1,3 @@
+use std::{thread:, mem};
```

Comment:
This test case demonstrates the case in which the new behavior is slightly worse and emits more spurious errors, and drops the `std::mem` `use`.